Gemfile

```diff
@@ -16,10 +16,10 @@ gem 'fastlane-plugin-sentry', '~> 1.0'
 #
 # gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: ''
 #
-# The '>= 13.3.1' aftre '~> 13.3' ensures that we resolve to any version compatible with 13.3 starting from 13.3.1
-# Using '~> 13.3.1' would constrain us to 13.3.2, 13.3.3, etc. without ever going up to 13.4
-# We need 13.3.1 because of a fix in screenshots generation we depend upon.
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 13.3', '>= 13.3.1'
+# The '>= 13.5.1' after '~> 13.5' ensures that we resolve to any version compatible with 13.5 starting from 13.5.1
+# Using '~> 13.5.1' would constrain us to 13.5.2, 13.5.3, etc. without ever going up to 13.6
+# We need 13.5.1 because of a fix in fetching issues / PRs with a fine-grained GitHub token
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 13.5', '>= 13.5.1'
```

Comment:
Same non-blocker, FYI note as in https://github.com/woocommerce/woocommerce-android/pull/14715#discussion_r2412106440 🙂